xdsl/dialects/memref.py

```diff
@@ -244,3 +226,7 @@ def get(sym_name: str | StringAttr,
                 "initial_value": initial_value,
                 "sym_visibility": sym_visibility
             })
+
+
+MemRef = Dialect([Load, Store, Alloc, Alloca, Dealloc, GetGlobal, Global],
```

Comment:
Should we actually keep the `MemRef` name format? Or should we switch to something like `memref_dialect`, since this is now an object, not a class?

Reply:
Would'nt we then also rename all the other dialects to `**_dialect`, for consistency? I am fine with doing it, but I am not sure how much information this name actually needs to contain. Does it make sense to mention the instantiated class in the variable name? Maybe @georgebisbas has some insights on what design makes more sense for our API.